bundles/org.openhab.binding.ipcamera/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -1986,6 +1986,14 @@
 				<advanced>true</advanced>
 			</parameter>
 
+			<parameter name="hasExternalAlarmInputs" type="boolean" groupName="Settings">
```

Comment:
Do not forget to adjust README to describe this new parameter and to add the two new entries (label and description) in the ipcamera.properties file.

Reply:
Good point, thanks.
I have added a new commit to address that.